eo-maven-plugin/src/main/java/org/eolang/maven/BinarizeMojo.java

```diff
@@ -54,9 +55,31 @@
     threadSafe = true,
     requiresDependencyResolution = ResolutionScope.COMPILE
 )
-@SuppressWarnings("PMD.LongVariable")
+@SuppressWarnings({"PMD.LongVariable", "PMD.StaticAccessToStaticFields"})
```

Comment:
@levBagryansky Maybe we add todo here? And think about how to avoid this PMD exclusion later. But it is up to you

Reply:
@Graur I think we should not avoid this exclusion because `StaticAccessToStaticFields` rule is for other cases.

Reply:
@levBagryansky maybe you can remove this suppression too